src/ReportManager/DetailsSection/index.js

```diff
@@ -46,14 +45,15 @@ const LOADER_COLOR = '#006cd9'; // Bright blue
 const LOADER_SIZE = 4;
 
 const DetailsSection = ({
-  formSchema,
-  formUISchema,
+  formSchema = null,
+  formUISchema = null,
   formValidator,
   isCollection,
   loadingSchema,
-  onFormChange,
+  onFormDataChange,
```

Comment:
I added the new callback to update the form data, and renamed the old one to `onLegacyFormChange` since we had to change a bit the logic of how we process the changes. They still do essentially the same thing.